src/components/zinzen-menu/UserChoiceZinZenMenu.jsx

```diff
@@ -4,12 +4,16 @@ import { useRecoilValue } from 'recoil';
 import { useTranslation } from 'react-i18next';
 
 import { darkModeState } from '../../store/DarkModeState';
+import { useNavigate } from 'react-router-dom';
```

Comment:
The convention to follow here is separate the package imports and file imports. So all package imports go together, then you add a blank line then add all the file imports then misc imports like `scss`, `css`, `json`, etc.